src/test/java/IntakeTest.java

```diff
@@ -41,4 +41,5 @@ public void testPush() {
     assertEquals(IntakeSubsystem.kOuttakeSpeed, dut.getSpeed(), kDelta);
   }
   // Add more @Test tagged methods to test other things...
+  */
```

Comment:
No, bad

Reply:
I had him remove it.  We were testing on swerve, which has mismatched motor id's, and a failure even if we disabled the subsystem in robot.java.

Can re-enable it and retest when the buddy balance hardware is moved to the competition bot, and if you want block merging until it tests clean.